examples/evkbmimxrt1060/defender/include/FreeRTOSConfig.h

```diff
@@ -58,7 +58,7 @@ extern void vLoggingPrintf( const char * pcFormat,
 #define configUSE_DAEMON_TASK_STARTUP_HOOK           1
 #define configCPU_CLOCK_HZ                           ( SystemCoreClock )
 #define configTICK_RATE_HZ                           ( ( TickType_t ) 1000 )
-#define configMAX_PRIORITIES                         ( 7 )
+#define configMAX_PRIORITIES                         ( 9 )

```

Comment:
Why are we changing the maximum priority from 7 to 9?

Reply:
After [commit in kernel](https://github.com/FreeRTOS/FreeRTOS-Kernel/commit/9efe10b805e3516974dd5eba2d61bfa071ff8392#diff-c396ff4650e7f6957c0fd758574eb5d39b71761458868c1053a1c979d8563276R917), it triggered an assertion because task priority is larger than max priorities. That's why I enlarge it here.